tests/test_jinja_macros.py

```diff
@@ -158,6 +159,25 @@ def test_score_macro(self, request_ctx):
             len(re.findall(r"""<span.*?\S*\d</span>""", rendered_html)) == 5
         ), "Should have 5 score values for radios"
 
+
+    def test_comment_macro(self, request_ctx):
+        rendered_html = render_template_string(
+            "{{commentBox(commentForm)}}",
+            commentBox=get_template_attribute("macros/comments_box.html", "commentBox"),
+            commentForm=CommentsForm()
+        )
+
+        # replacing new lines to more easily regex match the html
+        rendered_html = rendered_html.replace("\n", "")
+
+        assert re.search(
+            r"Add a comment</label>", rendered_html
+        ), "Title not found"
+
+        assert re.search(
+            r"Save comment</button>", rendered_html
+        ), "Title not found"
```

Comment:
Maybe should be "Save comment button not found"?

Reply:
Amended